web/themes/joinup/templates/field/field--fieldset-wrapper.html.twig

```diff
@@ -0,0 +1,65 @@
+{#
+/**
+ * @file
+ * Theme override for a field.
+ *
+ * To override output, copy the "field.html.twig" from the templates directory
+ * to your theme's directory and customize it, just like customizing other
+ * Drupal templates such as page.html.twig or node.html.twig.
+ *
+ * Instead of overriding the theming for all fields, you can also just override
+ * theming for a subset of fields using
+ * @link themeable Theme hook suggestions. @endlink For example,
+ * here are some theme hook suggestions that can be used for a field_foo field
+ * on an article node type:
+ * - field--node--field-foo--article.html.twig
+ * - field--node--field-foo.html.twig
+ * - field--node--article.html.twig
+ * - field--field-foo.html.twig
+ * - field--text-with-summary.html.twig
+ * - field.html.twig
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ * - label_display: The display settings for the label.
+ *
+ * @see template_preprocess_field()
+ */
+#}
+
+{% if label_hidden %}
+  {% if multiple %}
+    <div{{ attributes.addClass('fieldset') }}>
```

Comment:
Ok, here I'm confused. A bit earlier you make a [preprocess function for the global field](https://github.com/ec-europa/joinup-dev/pull/873/files#diff-b2909915abaa19a802586179a85c8677R647) and here you override a template for adding a class on the `attributes` if I understand correctly? Why adding a template for adding classes if you have already a preprocessor being able to manage existing variables?

Reply:
Agree that some changes are unclear. I wouldn't touch this changes because I am not sure whether this will not lead to a regression as this was made by Tom. I am quite sure that any possible duplicates have been made due to time pressure. This has to be reviewed.